pkg/adapter/adapter.go

```diff
@@ -72,6 +72,7 @@ type adapterConfig struct {
 	User           string `envconfig:"RABBITMQ_USER" required:"false"`
 	Password       string `envconfig:"RABBITMQ_PASSWORD" required:"false"`
 	Vhost          string `envconfig:"RABBITMQ_VHOST" required:"false"`
+	Predeclared    bool   `envconfig:RABBITMQ_PREDECLARED" required:"false"`
```

Comment:
```suggestion
	Predeclared    bool   `envconfig:"RABBITMQ_PREDECLARED" required:"false"`
```

Looks like we don't have any tests that cover Predeclared being true

Reply:
Thanks, i have added some tests there covering the whole cycle:
* Try to use it, fail (check the error)
* Declare a Queue in fakeServer
* Try again, it should work now